kafka_consumer/metadata.csv

```diff
@@ -2,3 +2,4 @@ metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation
 kafka.broker_offset,gauge,,offset,,Current message offset on broker.,0,kafka_consumer,broker offset,
 kafka.consumer_lag,gauge,,offset,,Lag in messages between consumer and broker.,-1,kafka_consumer,consumer lag,
 kafka.consumer_offset,gauge,,offset,,Current message offset on consumer.,0,kafka_consumer,consumer offset,
+kafka.estimated_consumer_lag_seconds,gauge,,second,,Lag in seconds between consumer and broker.,-1,kafka_consumer,consumer time lag,
```

Comment:
nit: I would add `kafka.estimated_consumer_lag` and avoid using the type in the metric's name because we already have the type in the metadata